### ISSUE TITLE
[CRITICAL] Filter is broken.

### ISSUE THREAD

**Issue by LebedevRI** [lsp-plugins/lsp-plugins#380]
This follows from https://github.com/wwmm/easyeffects/issues/2936,
but apparently this reproduces in a standalone environment for me:
(carla + pipewire, but also happens with carla + engine audio driver=ALSA; 192kHz)

https://github.com/lsp-plugins/lsp-plugins/assets/88600/96c5e3aa-8569-49c9-a598-d668723cdce3

**Comment by sadko4u** [comment]
Not reproducible under dummy JACK driver with 192 kHz sample rate.
LV2 plugins from `devel` branch loaded in Carla.
Have you tried to reproduce the problem with `devel` branch?
Also, what's your I/O buffer size?

**Comment by LebedevRI** [comment]
> LV2 plugins from `devel` branch loaded in Carla.

debian unstable package here:
```
ii  lsp-plugins                                   1.2.14-1                                                      amd64        LSP (Linux Studio Plugins) audio-plugins (metapackage)
```

> Have you tried to reproduce the problem with `devel` branch?

I have never build lsp-plugins, so no.

> Also, what's your I/O buffer size?

`8192`, and CPU is idle, DSP load is ~10%.

**Comment by sadko4u** [comment]
Here's the demonstration:
https://github.com/lsp-plugins/lsp-plugins/assets/12834464/39720281-3362-4a60-a86a-18dd046f2f10

**Comment by sadko4u** [comment]
With buffer size 8192 the problem seems to be reproducible.

**Comment by LebedevRI** [comment]
Hurray :)

**Comment by sadko4u** [comment]
As a temporary workaround, you can reduce your Pipewire buffer size at this moment.

**Comment by sadko4u** [comment]
It seems that equalizer smoothing mode introduced for avoiding pops and clicks when changing parameters of filters like Frequency, Gain and Q factor worked improperly.

I can not reproduce the problem any more after these commits:
https://github.com/lsp-plugins/lsp-plugins-para-equalizer/commit/a32854c56be2d91ce61a50c39fa6c11943bbf196
https://github.com/lsp-plugins/lsp-plugins-filter/commit/652fa15e03958bc017f9b8b3b6de3019bbf26929

**Comment by LebedevRI** [comment]
Thank you!

So basically the same thing as https://github.com/lsp-plugins/lsp-plugins/issues/348
Might be worth it to look for similar issues in the entire codebase...

**Comment by sadko4u** [comment]
No, that was a different thing related to buffer overflow. Here we did wrong approximation of filter parameters.
Symptoms are similar but the reason is different.

**Comment by sadko4u** [comment]
No, this is only related to Parametric Equalizer and Filter plugins since only these plugin series provide filter smoothing feature.

**Comment by sadko4u** [comment]
Fixed in 1.2.15.

**Comment by LebedevRI** [comment]
@sadko4u thank you!
